pyproject.toml

```diff
@@ -53,6 +69,16 @@ litellm = 'litellm:run_server'
 flake8 = "^6.1.0"
 black = "^23.12.0"
 pytest = "^7.4.3"
+mypy = "^1.8.0"
+pytest-async = "^0.1.1"
```

Comment:
This needs to be replaced by `pytest-asyncio` (a different package - you had this in the manual installs before). This is why you're still getting warnings in Circle CI logs:

```
PytestUnknownMarkWarning: Unknown pytest.mark.asyncio - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.asyncio
```